hw_11/HW_1_Sotnikov.py

```diff
@@ -0,0 +1,368 @@
+#!/usr/bin/env python3
+
+# 1 Chat-message-user
+from datetime import datetime
+
+## Creation class chat
+class Chat(list):
+    def __init__(self, container):
+        self.chat_history = []
+
+
+    def show_last_message(self, number=0):
+        last_message = self.chat_history[number]
+        print(f"User info: {last_message.show('user')}\n\
+        Date: {last_message.show('date')}\n\
+        Time: {last_message.show('time')}\n\
+        Text of the message: {last_message.show('text')}")
+
+
+    def get_history_from_time_period(self, first_date=None, second_date=None):   # Функция работает криво
+        start_point = None
+        final_point = None
+        Flag_start = True
+
+        if first_date != None or second_date != None:
```

Comment:
`None` проверяется через `is`

```suggestion
        if first_date is not None or second_date is not None:
```